gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoMessageRequestFormatter.java

```diff
@@ -136,7 +156,8 @@ public ProtoMessageRequestFormatter<RequestT> build() {
           queryParamsExtractor,
           rawPath,
           PathTemplate.create(rawPath),
-          pathVarsExtractor);
+          pathVarsExtractor,
+          rawAdditionalPaths.stream().map(PathTemplate::create).collect(Collectors.toList()));
```

Comment:
Can we make it a ImmutableList? `Collectors.toList()` is not guaranteed to be immutable.

Reply:
ImmutableList collectors are not part of a more lightweight versions of guava (androind one does not have them), so to keep better portability I believe it is better to stick to standard library collectors.